examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<ul><li><s>Implimenting</s> <strong>Implementing</strong> the censored regression</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>fixed</p>